src/kaleidoscope_internal/sketch_exploration/keymap_exploration.h

```diff
@@ -214,7 +214,7 @@ extern void pluginsExploreSketch();
 // exploreSketch<_Sketch>(...)-hook.
 //
 #define _INIT_KEYMAP_EXPLORATION                                               \
-  namespace kaleidoscope {                                                     \
+  namespace kaleidoscope { /* NOLINT(build/namespaces) */                      \
```

Comment:
This namespace is inside a macro, we don't need to care whether it is fully declared prior to evaluating the macro. Tell cpplint to not warn about it.